kinetic/distribution.yaml

```diff
@@ -5448,14 +5448,23 @@ repositories:
       url: https://github.com/MarvelmindRobotics/marvelmind_nav-release.git
       version: 1.0.8-0
   mav_comm:
+    doc:
+      type: git
+      url: https://github.com/ethz-asl/mav_comm.git
+      version: 3.3.0
```

Comment:
this should be a branch rather than a tag, I'm guessing `master` is this is what is [used for indigo](https://github.com/ros/rosdistro/blob/79242ea09a3703b0f715b5742eea3d7861e103c3/indigo/distribution.yaml#L6952).

Same below for the source section